dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -290,31 +290,36 @@ namespace dwa_local_planner {
     std::vector<geometry_msgs::PoseStamped> front_global_plan = global_plan_;
     double angle_to_goal = atan2(goal_pose.pose.position.y - pos[1], goal_pose.pose.position.x - pos[0]);
 
-    // reduce crop forward_point_distance such that the robot can reach the goal pose
-    // without its nose entering space considered as occupied by obstacles
-    double forward_point_distance = 0;
+    constexpr double MIN_GOAL_DIST_SQ = 0.7;
```

Comment:
F: This value seems arbitrary and it is hidden in the middle of the cpp file. 
It should be in the header (static) or, better, a parameter.